include/envelope.h

```diff
@@ -78,15 +78,13 @@ class Envelope {
 
 	bool ClampSample(int &sample, int next_edge);
 
-	void Apply(bool is_stereo, bool is_interpolated, int prev[], int next[]);
+	void Apply(bool is_stereo, int frame[]);
 
 	void Skip([[maybe_unused]] bool is_stereo,
-	          [[maybe_unused]] bool is_interpolated,
-	          [[maybe_unused]] int prev[],
-	          [[maybe_unused]] int next[])
+	          [[maybe_unused]] int frame[])
 	{}
 
-	using process_f = std::function<void(Envelope &, bool, bool, int[], int[])>;
+	using process_f = std::function<void(Envelope &, bool, int[])>;
```

Comment:
I believe I've done the correct thing by removing the interpolation path from `envelope`, but I haven't figured out a way how to test it easily. I hope I haven't broken it, @kcgen! 😊

Reply:
Looks good, and it's still working.
It's only active for the initial audio that hits a channel, and only if the waveform grows too quickly, which is good for startup pop-suppression.  I fired up Shamus, and here catches the initial square wave:

![2022-05-20_09-54](https://user-images.githubusercontent.com/1557255/169576321-994e0a0f-6628-4773-8cc8-1836e38084e2.png)